yapapi/golem.py

```diff
@@ -28,6 +28,37 @@
 
 
 class Golem(_Engine):
+    """
+    The main entrypoint of the Golem's high-level API.
```

Comment:
```suggestion
    """The main entrypoint of the Golem's high-level API.
```
Flake 8 docstrings plugin, if we ever enable it, would require us to change that. That would be also consistent with other multi-line docstrings e.g. in `yapapi.executor`.